CONTRIBUTING.md

```diff
@@ -4,70 +4,68 @@
 
 ### If your problem is with...
 
- - [ ] automated builds
- - [ ] your account on the [Docker Hub](https://hub.docker.com/)
- - [ ] any other [Docker Hub](https://hub.docker.com/) issue
+ - automated builds
+ - your account on the [Docker Hub](https://hub.docker.com/)
+ - any other [Docker Hub](https://hub.docker.com/) issue
 
-Then please do not report your issue here - you should instead report it to [https://support.docker.com]
+Then please do not report your issue here - you should instead report it to [https://support.docker.com](https://hub.docker.com/)
```

Comment:
@dmp42 Looks like a mistake. @thaJeztah Does support.docker.com not work for you?